php/php.editor/src/org/netbeans/modules/php/editor/verification/WrongOrderOfArgsHint.java

```diff
@@ -150,12 +150,18 @@ public String getDescription() {
         @Override
         public void implement() throws Exception {
             EditList edits = new EditList(doc);
-            OffsetRange offsetRange = getOffsetRange();
-            StringBuilder sb = new StringBuilder();
-            for (ParameterElement param : functionDeclarationInfo.getParameters()) {
-                sb.append(param.asString(OutputType.COMPLETE_DECLARATION)).append(", "); //NOI18N
+            List<FormalParameter> originalParameters = node.getFormalParameters();
+            List<ParameterElement> parameters = functionDeclarationInfo.getParameters();
+            assert originalParameters.size() == parameters.size();
```

Comment:
Nitpick: maybe better as:

```
assert originalParameters.size() == parameters.size() : originalParameters.size() + " != " + parameters.size();
```

Or similarly, of course.

Reply:
Will fix it. Thank you!